habitat/datasets/rearrange/configs/pick/counter.yaml

```diff
@@ -0,0 +1,178 @@
+---
+dataset_path: "data/replica_cad/replicaCAD.scene_dataset_config.json"
```

Comment:
Not really relevant to this PR but I get the sense that configurations and datasets/scenes are often referred as their paths. I think this is going to be maybe hard to maintain on the long-term. This forces users to hard code paths at a lot of places. I think having some static registry of keys to paths might be a better approach. This way if you want to move an asset, you would just need to change it at one place (and you could also populate this registry every time an asset is downloaded).